### DISCUSSION TITLE
Could only predict one instance

### DISCUSSION THREAD

**Discussion by LianhaoYin** [talmolab/sleap#1319] [Help!] [answered]
# Bug description

The videos are with multiple animals in each frame. I labeled multiple frame and trained using bottom up method, however, the prediction only predict one instance in one frame. 

## Expected behaviour
It should predict multiple instance in one frame since there are multiple animals

## Actual behaviour
It only predict one instance in one frame. 

## Your personal set up
I'm using sleap main branch
sleap-label 

ubuntu 20.04
sleap main. python 3.8
- OS:

- Version(s):

- SLEAP installation method (listed [here](https://sleap.ai/installation.html#)):
  - [ ] [Conda from package](https://sleap.ai/installation.html#conda-package)
  - [ ] [Conda from source](https://sleap.ai/installation.html#conda-from-source)
  - [ ] [pip package](https://sleap.ai/installation.html#pip-package)
  - [ ] [Apple Silicon Macs](https://sleap.ai/installation.html#apple-silicon-macs)

<details><summary>Environment packages</summary>

```
# paste output of `pip freeze` or `conda list` here
```

</details>

<details><summary>Logs</summary>

```
# paste relevant logs here, if any
```

</details>

**Comment by roomrys** [comment]
Hi @LianhaoYin,

This sounds like it is a problem with either the labeling, the model used, or the model hyperparameters.

Some questions that would help diagnose the problem:
1. How many frames did you label? 
2. Did you label both instances in every frame? 
3. Which model did you use (Single Instance would only predict a single instance)? 
4. Could you share a screenshot of the Training Pipeline Model Configuration? 
5. Were there any logs?  
6. Which installation method did you use? 
7. Which version of SLEAP are you on? 
8. Which OS?

Thanks,
Liezl

**Comment by LianhaoYin** [comment]
Resolved. Not relevant anymore. Thanks @roomrys

**Reply by roomrys** [reply]
@LianhaoYin,

Thanks for following up! Do you mind mentioning your solution (i.e hyperparameter tuning, model selection, etc.)?

Thanks,
Liezl

**Reply by LianhaoYin** [reply, marked as answer]
I can not reproduce the error right now, but potentially model selection is the problem. Using top-down model helps.
